packages/react-native-windows-init/src/Cli.ts

```diff
@@ -87,13 +89,19 @@ const EXITCODE_NO_LATEST_RNW = 8;
 const EXITCODE_NO_AUTO_MATCHING_RNW = 9;
 const EXITCODE_INCOMPATIBLE_OPTIONS = 10;
 
-function reactNativeWindowsGeneratePath() {
-  return require.resolve('react-native-windows/local-cli/generate-windows.js', {
-    paths: [process.cwd()],
-  });
+function requireGenerateWindows(): typeof generateWindows  {
+  try {
+    // Try the path for 0.63+
+    return require('react-native-windows/local-cli/lib-commonjs/generate-windows');
```

Comment:
I might need to add .default to the end of this one. Think I still need to use ES6 exports for this if I want to also export the type?

Reply:
Going to just avoid default export for the new path since the rules are always hazy and IIRC flag dependent.